app/views/subscription_charges/new.html.erb

```diff
@@ -52,7 +52,7 @@
         <%= recaptcha_tags %>
 
         <div class="actions">
-          <%= form.submit "Submit Payment", class: 'button primary' %>
+          <%= form.submit "Enroll Membership for #{number_to_currency(@current_plan.amount).gsub(/\.00$/, "")}/mo", class: 'button primary', id: "submit-payment" %>
```

Comment:
@castrolem what about using `Math.round` instead of gsub?

Reply:
`Math.round` could be cumbersome in the future if the pricing involves cents. If needed to show cents, later on. Only the amount with no cents should be hidden.